readme.md

```diff
@@ -73,3 +73,55 @@ bot.once('spawn', () => {
 * Dynamic harvest/mining paths
 * Sprint
 * Actual move speed per block
+
+## API
+Considering there are a lot of deep changes that are being worked on, it could take some time before it's done
+
+Also, **for now**, there is only the `pathfinder` module, `movements` and `goals` still need to be done
+
+
+# Functions:
+
+### bot.pathfinder.bestHarvestTool(block)
+Returns the best harvest tool in the inventory for the specified block
+ * `Return` - ?
+ * `block` - Block instance
+
+### bot.pathfinder.getPathTo(movements, goal, done, timeout)
+Returns a Path instance
+ * `Return` - Path instance
+ * `movments` - Movements instance
+ * `goal` - Goal instance
+ * `done` - ?
+ * `timeout` - ?
+
+### bot.pathfinder.setGoal(Goal, dynamic)
+ * `goal` - Goal instance
+
+### bot.pathfinder.setMovements(movements)
+ * `movments` - Movements instance
```

Comment:
similar to the comment about the `Goal` parameter in `bot.pathfinder.setGoal` it would be nice to make a section detailing how you can customize movements using the `Movements` class